src/components/SegmentedProgressBar/SegmentedProgressBar.jsx

```diff
@@ -1,34 +1,67 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import _ from 'lodash';
+import dispatchAnalyticsEvent from '../../helpers/analytics';
 
 /**
  * Create a segmented progress bar for multi-page forms.
  *
  * @param {number} current - The index of the current chapter
  * @param {number} total   - The total number of chapters in the form
  */
-export default function SegmentedProgressBar({ current, total }) {
-  return (
-    <div
-      className="progress-bar-segmented"
-      role="progressbar"
-      aria-valuenow={current}
-      aria-valuemin="0"
-      aria-valuemax={total}
-      tabIndex="0"
-      aria-label={`Step ${current} of ${total}`}
-    >
-      {_.range(total).map(step => (
-        <div
-          key={step}
-          className={`progress-segment ${
-            current > step ? 'progress-segment-complete' : ''
-          }`}
-        />
-      ))}
-    </div>
-  );
+export default class SegmentedProgressBar extends React.Component {
```

Comment:
Same comment here.

Reply:
..and this one...

https://github.com/department-of-veterans-affairs/component-library/blob/bba37848873b57e8108bc886ff7934085836661a/src/components/SegmentedProgressBar/SegmentedProgressBar.jsx